templates/zerver/landing_nav.html

```diff
@@ -16,26 +16,89 @@
         </a>
         <ul>
             <div class="exit">&times;</div>
-            <li data-on-page="hello">
-                <a href="/hello/">Home</a>
-            </li>
-            <li data-on-page="why-zulip">
-                <a href="/why-zulip/">Why Zulip</a>
-            </li>
-            <li data-on-page="features">
-                <a href="/features/">Features</a>
-            </li>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Product<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list product-options">
+                    <li>
+                        <a href="/hello">Home</a>
+                    </li>
+                    <li>
+                        <a href="/why-zulip">Why Zulip</a>
+                    </li>
+                    <li>
+                        <a href="/features/">Features</a>
+                    </li>
+                    <li>
+                        <a href="/security/">Security</a>
+                    </li>
+                    <li>
+                        <a href="/integrations/">Integrations</a>
+                    </li>
+                    <li>
+                        <a href="/api/">REST API</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Solutions<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list solutions-options">
+                    <li>
+                        <a href="/for/companies/">Companies</a>
+                    </li>
+                    <li>
+                        <a href="/for/open-source/">Open source projects</a>
+                    </li>
+                    <li>
+                        <a href="/for/working-groups-and-communities/">Working groups and communities</a>
+                    </li>
+                </ul>
+            </div>
             <li data-on-page="plans">
-                <a href="/plans/">Plans</a>
+                <a href="/plans/">Pricing</a>
             </li>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Resources<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list resources-options">
+                    <div class="dropdown-column">
+                        <li class="nav-header">For users</li>
+                        <li>
+                            <a href="/help/getting-started-with-zulip">Getting started</a>
+                        </li>
+                        <li>
+                            <a href="/help">Help center</a>
+                        </li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/latest/contributing/chat-zulip-org.html" target=”_blank”>
```

Comment:
Our policy is generally that all links with `target=_blank` need `rel="noopener noreferer"` as well.